ami/trainers/i_jepa_latent_visualization_trainer.py

```diff
@@ -0,0 +1,217 @@
+import time
+from functools import partial
+from pathlib import Path
+from typing import Literal
+
+import torch
+import torch.nn.functional as F
+import torchvision.transforms.v2.functional as torchvisF
+import torchvision.utils
+from torch import Tensor
+from torch.optim import Optimizer
+from torch.utils.data import DataLoader, Dataset
+from typing_extensions import override
+
+from ami.data.buffers.buffer_names import BufferNames
+from ami.data.buffers.random_data_buffer import RandomDataBuffer
+from ami.data.interfaces import ThreadSafeDataUser
+from ami.models.bool_mask_i_jepa import BoolMaskIJEPAEncoder
+from ami.models.i_jepa import IJEPAEncoder
+from ami.models.i_jepa_latent_visualization_decoder import (
+    IJEPALatentVisualizationDecoder,
+)
+from ami.models.model_names import ModelNames
+from ami.models.model_wrapper import ModelWrapper
+from ami.tensorboard_loggers import StepIntervalLogger
+
+from .base_trainer import BaseTrainer
+
+
+class IJEPALatentVisualizationTrainer(BaseTrainer):
```

Comment:
Decoderであることを強調したい(あるか分からないけど今後`IJEPALatentVisualizationDiffusionTrainer`みたいなのが出てくるのを想定)
```suggestion
class IJEPALatentVisualizationDecoderTrainer(BaseTrainer):
```

Reply:
OK